packages/react-native-bpk-component-dialog/src/NativeDialog.ios.js

```diff
@@ -26,7 +26,7 @@ import {
   processColor,
 } from 'react-native';
 import type EmitterSubscription from 'react-native/Libraries/vendor/emitter/EmitterSubscription';
-import isNil from 'lodash/isNil';
+import _ from 'lodash';
```

Comment:
Should we really be importing all of `lodash` like this over specific imports for tree shaking?